dtfs-central-api/src/types/tfm/teams.ts

```diff
@@ -0,0 +1,34 @@
+export type Team = { email: string } & (
+  | {
+      id: 'UNDERWRITER_SUPPORT';
+      name: 'Underwriter support';
```

Comment:
True, but in terms of the type we don't really care what the name is, just that it's a string. There is no benefit to including these in the type as there is no logic based off of it (at least there shouldn't be), so we don't care if they change. The id we DO care about because we base logic of it and need to know if it changes because it will break things.